README.md

```diff
@@ -17,22 +19,30 @@ sudo dnf install \
     lcov \
     libasan \
     libbpf-devel \
+    libnl3-devel \
     libubsan \
     python3-breathe \
     python3-furo \
     python3-sphinx \
     pkgconf
 
 # Build bpfilter
-cmake -S $BPFILTER_SOURCES -B $BPFILTER_BUILD
-make -C $BPFILTER_BUILD
-make -C $BPFILTER_BUILD test
+cmake -S $SOURCES_DIR -B $BUILD_DIR
+make -C $BUILD_DIR
+make -C $BUILD_DIR test
+
+# Build a custom version of nftables and iptables to use with bpfilter
+make -C $BUILD_DIR nftables iptables
```

Comment:
This is fine, but FYI one can also use `cmake -Gninja` (and environments do that by default); I have no idea if it works with this codebase, but it might be worth documenting either way.

Reply:
I've added a one line fix to allow Ninja to be used, and updated the documenation. I'll kept the README.md as short as possible as this is now documented in the full instructions.